src/go/k8s/internal/testutils/gorutines.go

```diff
@@ -0,0 +1,24 @@
+package testutils
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// TGo is a helper for ensuring that goroutines spawned in test cases are
+// appropriately shutdown.
+func TGo(t *testing.T, ctx context.Context, fn func(context.Context) error) {
```

Comment:
nit: I'd drop the `T` from the name as the package name provides the context that `T` did. `testutils.Go` vs `testutils.TGo`

Reply:
THX, I will do that.